geos.c

```diff
@@ -19,6 +19,28 @@ void go_geos_error_handler(const char *fmt, ...) {
     va_end(ap);
 }
 
+void go_geos_LinearRingToFlatPoints (GEOSContextHandle_t handler, double out[], size_t n_out, const GEOSCoordSequence *seq) {
+    int i;
+    double *j = out;
+    for (i = 0; i < n_out; i++) {
+        GEOSCoordSeq_getXY_r(handler, seq, i, j, j + 1);
+        j++;
```

Comment:
Why not `+= 2`? Using twice the same `++` operator looks like a mistake

Reply:
Fixed